blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -114,6 +111,24 @@ public void removeAllMetricNames() {
         log.info("[TRACKER] All metric names removed.");
     }
 
+    public Set<String> getMetricNames() {
+        return this.metricNames;
```

Comment:
This is not the same issue, because there is no parameter with the same name in the method. We should just remove the "this." part.